src/MergePlugin.php

```diff
@@ -35,20 +35,26 @@
 /**
  * Composer plugin that allows merging multiple composer.json files.
  *
- * When installed, this plugin will look for a "merge-patterns" key in the
+ * When installed, this plugin will look for a "merge-plugin" key in the
```

Comment:
Nice catch, that was the original key when I first started writing this and hadn't added a whole block of settings. We should fix up other docs here in another pull request or ditch it all together and just focus on keeping the README up to date.